config/playground/helm/dev/dashboards.yaml

```diff
@@ -0,0 +1,203 @@
+# Copyright OpenSearch Contributors
```

Comment:
1. Let's elaborate how these two files are generated, i.e follow any templates
2. Any tests did for these two files?

Reply:
1. Right so the configuration changes for OS start on lines 311 to 918 of `master.yaml`. The only thing modified from the default OS configuration is the security configurations, and more specifically, the only 3 `.yml` files I've changed are `config.yml`, `roles.yml`, and `roles_mapping.yml`

2. I only manually tested some aspects of the site, like checking to see if the site doesn't allow me to create data or change settings. Are there any tools that can automate the process of checking for read-only access?

Reply:
> 1. Right so the configuration changes for OS start on lines 311 to 918 of `master.yaml`. The only thing modified from the default OS configuration is the security configurations, and more specifically, the only 3 `.yml` files I've changed are `config.yml`, `roles.yml`, and `roles_mapping.yml`. As for OSD, there is only one change made to `dashboards.yaml` on lines 74 to 87, and only one option was enabled: `opensearch_security.auth.anonymous_auth_enabled: true`.

Thank you for providing context; please include these details in the PR description as well. Is this configuration aimed at the production environment? If that's the case, let's double-check that all other security settings are set to the recommended levels.

> 2\. I only manually tested some aspects of the site, like checking to see if the site doesn't allow me to create data or change settings. Are there any tools that can automate the process of checking for read-only access?

No, I think manual testing is sufficient; let's include testing details in PR descriptions.

Reply:
> Thank you for providing context; please include these details in the PR description as well. Is this configuration aimed at the production environment? If that's the case, let's double-check that all other security settings are set to the recommended levels.

I believe I will be making changes to a dev environment but eventually this will go to prod. Thus, when the switch happens I'll double check permissions.